docs/plugins/known-plugins.asciidoc

```diff
@@ -60,7 +60,6 @@ This list of plugins is not guaranteed to work on your version of Kibana. Instea
 [float]
 === Other
 * https://github.com/nreese/kibana-time-plugin[Time picker as a dashboard panel] Widget to view and edit the time range from within dashboards.
-* https://github.com/sw-jung/kibana_notification_center[Notification Center] (sw-jung) - for better experience of notifier toasts.
```

Comment:
I removed the notification center plugin, since it only worked for `Notifier` notifications.